### PR TITLE
Outsourced MongoDB connection string to environment variable

### PR DESCRIPTION
**Pull request by electron2302**
Just to lines, but it solves #6 and gives the user the freedom to change the DB connection to anything their situation might require.

How would you like it to be documented thou? 
Should I put it as a comment in the compose file?

### CODE REVIEW

File: server/src/database/index.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ const connect = async () => {
   while (1) {
     try {
       logger.debug('Trying to connect to database');
-      await mongoose.connect('mongodb://mongo:27017/your_spotify', { useNewUrlParser: true });
+      await mongoose.connect(process.env.MONGO_ENDPOINT, { useNewUrlParser: true });
```
Comment (Yooooomi) [inline server/src/database/index.js:14]: Maybe write `process.env.MONGO_ENDPOINT || 'mongo'` here so that the user is not compelled to define it

**Comment by Yooooomi** [comment]
Wrote a comment on your implementation. As for the documentation, yes, writing it in the docker-compose file, commented out, wouldbe a great idea

**Comment by electron2302** [comment]
> Wrote a comment on your implementation.
Where ?

File: server/src/database/index.js (hunk 1 above)
Comment (electron2302) [reply server/src/database/index.js:14]: Since the environment variable has a default value this would be dead code.

Comment (Yooooomi) [reply server/src/database/index.js:14]: Ok !

Comment (electron2302) [reply server/src/database/index.js:14]: Ok, haven’t thought about that

```suggestion
      const fallbackConnection = 'mongodb://mongo:27017/your_spotify'
      await mongoose.connect(process.env.MONGO_ENDPOINT || fallbackConnection, { useNewUrlParser: true });
```

**Pull request merged**